__tests__/app.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+const app = require("../app");
+
+beforeEach(() => {
+    return seed(data);
+    
+})
+afterAll(() => {
+    db.end();
+})
+
+describe("/api/topics endpoint", () => {
+
+    describe("GET all", () => {
+        it("should return status - 200 ok", () => {
+        return request(app).get("/api/topics").expect(200);
+    });
+
+    it("should return the list of expected topics from the data file.", () => {
+        return request(app).get("/api/topics").expect(200).then((res) => {
+            const { topicData } = data;
+            expect(res.body).toEqual({ topics: topicData})
```

Comment:
we need to be careful when doing this as the data coming back from the database won't always looks like the data that is inserted INTO the database. 

Please amend this so that you are checking what is coming back from the database more specifically

e.g. - what properties are you expecting to see/type of etc